ecosystem/indexer-grpc/indexer-grpc-in-memory-cache-benchmark/Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [package]
 name = "aptos-indexer-grpc-in-memory-cache-benchmark"
+version = "0.1.0"
```

Comment:
Is this relevant change?

Reply:
Well what I found out was that `cargo fix` would complain about this not having a version but strangely `cargo build` does not. Decided to fix it as `cargo fix` can be a really helpful command for removing unused imports.